Week-2/assignment-2.1/styles/index.css

```diff
@@ -0,0 +1,249 @@
+html {
+    box-sizing: border-box;
+    overflow-y: scroll;
+    overflow-x: hidden;
+    height: 100%;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    display: flex;
+    flex-direction: column;
+    padding: 2em;
+    align-items: center;
+    font-family: Georgia, serif;
+    margin: 0;
+    padding: 0;
+    background: #f1f2f2;
+}
+
+.column {
+    display: flex;
+    flex-direction: column;
+}
+
+.row {
+    display: flex;
+}
+
+@media only screen and (max-width: 1215px) {
+    header {
+        position: relative !important;
+    }
+
+    section {
+        justify-content: center;
+    }
+
+    .flex-item-right,
+    .flex-item-left {
+        flex: 100rem !important;
```

Comment:
you should try to place media queries after the normal css so that you dont have to put !important,